frontend/src/metabase/components/LastEditInfoLabel.js

```diff
@@ -32,19 +32,21 @@ LastEditInfoLabel.propTypes = {
   className: PropTypes.string,
 };
 
-function formatEditorName(firstName, lastName) {
-  const lastNameFirstLetter = lastName.charAt(0);
-  return `${firstName} ${lastNameFirstLetter}.`;
+function formatEditorName(lastEditInfo) {
+  //  https://user-images.githubusercontent.com/1937582/172189537-0eb7f620-559d-4537-8467-5e1ea5bab78a.png
+  const name = [lastEditInfo.first_name, lastEditInfo.last_name]
+    .join(" ")
+    .trim();
+
+  return name || lastEditInfo.email;
```

Comment:
Following [the discussion on Slack](https://metaboat.slack.com/archives/C03E8MQJZBM/p1654766630306639?thread_ts=1654759255.159039&cid=C03E8MQJZBM), this special one-off place should also be replaced with `common_name` but since `common_name` doesn't exist on `last-edit-info`, I need to manually fall back to email.

#### Edited by you
<img src="https://user-images.githubusercontent.com/1937582/172189537-0eb7f620-559d-4537-8467-5e1ea5bab78a.png" />

#### Edited by others
![image](https://user-images.githubusercontent.com/1937582/172850763-dee5745b-8af2-45e8-b9e5-5d9c03b51b8d.png)